.gitignore

```diff
@@ -1,22 +1,23 @@
 .idea/
 .vs/
 
+.DS_Store
 *.class
 *.iml
+*.zip
+
+ft_temp/
 /target/
 .gradle/
-
+build/
 repos/
 logs/
 out/
-ft_temp/
-*.zip
 
-build/
-frontend/build
 node_modules
-package-lock.json
```

Comment:
You may want to leave the `package-lock.json` line in for now since removing the frontend package-lock.json should not be in this PR's scope.

Reply:
Alright, I will re-add this and add an exception to exclude `frontend/cypress/package-lock.json`

Reply:
Ideally, this is merged before #1863, so that in #1863 we can remove the exception of `!frontend/cypress/package-lock.json`